Few Fixes

SriHarshaGajavalli opened the pull request:
Gemfile is updated to ensure successful installation of phantomJS for running the tests.

While running the test cases, command_injection_spec is going to pending, I have fixed it which ensures successful run all the time.

A justification for this PR
1. To run tests, phantomJS is needed and there is a need for a gem installation. Hence I have added and sent a PR of the updated Gemfile to make developers life easier.

2. Coming to the change in command_injection_spec, this particular test is not succeeding all the time. This particular issue is fixed with the proposed changes

cktricky commented:
Hey @SriHarshaGajavalli 👋 - Thanks for submitting a PR. Can you provide some context as to the changes, their purpose, etc.?

SriHarshaGajavalli commented:
Hello @cktricky - I have updated the description

jasnow commented:
@SriHarshaGajavalli - How does this PR impact the phantomJS installation instructions in the README file?

jasnow commented:
Why do you want the poltergeist gem twice in the Gemfile file?

SriHarshaGajavalli commented:
@jasnow poltergeist gem is already existing but it is failing to get installing sometime. Hence I have added. But it is because of some issues on the local machine. However, I have removed it now.

Coming to the change command_injection_spec, everytime a test is running, it is creating a file and the test is going to pending status instead of failing (which is a successful run)

SriHarshaGajavalli commented:
It isn't failing. Instead, it's successful execution leads to the creation of a file which in turn results in the test case status pending

jasnow commented:
```
It isn't failing. Instead, it's successful execution leads to the 
creation of a file which in turn results in the test case status pending
```
With or without commenting out line 17?

SriHarshaGajavalli commented:
With commenting, test case executes successfully (which means actually failed). If we uncomment the line, the test case is going to pending status.

I guess there could be bit of confusion with the concept itself

jasnow commented:
@SriHarshaGajavalli -  As you mention above:
```
Coming to the change in command_injection_spec, this particular test is not succeeding all the time. This particular issue is fixed with the proposed changes
```
Do you know why the command_injection_spec  is succeeding sometimes and not others?

SriHarshaGajavalli commented:
@jasnow : Because of the above mentioned reason, it never succeeded for me

jasnow commented:
Could not reproduce the problem so rejecting the fix.

jasnow commented:
@SriHarshaGajavalli  - You are welcome to reopen this issue when you have more data to share.
